diesel/src/query_builder/functions.rs

```diff
@@ -124,3 +124,10 @@ pub fn select<T>(expression: T) -> Select<SelectStatement<()>, T> where
 {
     SelectStatement::simple(()).select(expression)
 }
+
+/// Creates an insert statement. Will insert default values in the table. This
+/// function is not exported by default. As with other commands, the resulting
+/// query can return the inserted rows if you choose.
```

Comment:
True. We should think of something to make it easier to write these doc tests (and maybe also write it down somewhere)